spec/controllers/lending_policy_controller_spec.rb

```diff
@@ -46,15 +46,28 @@
       context 'when the patron is eligible for wage garnishment' do
```

Comment:
Can we also add a spec for when patron is not eligible for wage garnishment?

Reply:
It's there, it just doesn't show up in the diff since it was there already